schema.sql

```diff
@@ -57,4 +57,11 @@ ADD FOREIGN KEY (species_id) REFERENCES species(id);
 
 ALTER TABLE animals
 ADD COLUMN owner_id INT,
-ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
\ No newline at end of file
+ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX index_animals_vets_visits ON visits (animal_id, vets_id);
+
+CREATE INDEX index_email_owners ON owners (email);
```

Comment:
- [ ] [REQUIRED] Good job so far! You should create `INDEX` for each property(`animal_id`, `vets_id` and `email`) separate `CREATE INDEX` selector and also analyze screenshots should be `3` on your `PR` summary, please implement it 🙏🏼 

- For Example =>
```sql
   CREATE INDEX ON visits (animal_id);
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);
```

![image](https://user-images.githubusercontent.com/63915024/214323364-2f377931-04ba-4ef8-9494-f40c186f4dcc.png)